packages/containers/src/ComponentForm/ComponentForm.component.js

```diff
@@ -22,6 +22,7 @@ const TO_OMIT = [
 
 export const DEFAULT_STATE = new Map({
 	dirty: false,
+	initialState: {},
```

Comment:
Wouldn't it be less confusing to call this `initialData` ?

Reply:
in UIForm it is already called initialState didn't wan't to create discrepancies now.